src/components/Footer/Footer.js

```diff
@@ -0,0 +1,95 @@
+import React from "react";
+import {
+  FooterContainer,
+  FooterLinkItems,
+  FooterLinksContainer,
+  FooterLinksWrapper,
+  FooterWrap,
+  FooterLinkTitle,
+  FooterLink,
+  SocialMedia,
+  SocialMediaWrap,
+  Logo,
+  WebsiteRights,
+  SocialIcons,
+  SocialIconLink,
+} from "./FooterElements";
+import { IconContext } from "react-icons";
+import {
+  FaFacebook,
+  FaInstagram,
+  FaYoutube,
+  FaTwitter,
+  FaLinkedin,
+} from "react-icons/fa";
+import logo from "../../assets/images/icon2.svg";
+import { Link } from "react-router-dom";
+
+export const Footer = () => {
+  return (
+    <FooterContainer>
```

Comment:
This particuale section of code could be rewitten to break up the components inside of Footer in a more elegant manner.